posthog/async_migrations/migrations/0004_replicated_schema.py

```diff
@@ -151,9 +152,11 @@ def replicated_table_operations(self, table: TableMigrationData):
             rollback=f"DROP TABLE IF EXISTS {table.tmp_table_name}",
         )
 
-        if table.kafka_table_name is not None:
+        for i in range(len(table.kafka_table_names)):
```

Comment:
`for create_kafka_table, kafka_table_name in zip(table.create_kafka_tables table.kafka_table_names)` 

Range is _often_ a sign that something has gone wrong